src/components/Layout/Navbar.tsx

```diff
@@ -1,17 +1,185 @@
+      {({ open }) => (
+        <>
+          <div className="mx-auto max-w-screen px-2 sm:px-6 lg:px-8 bg-white border-b-2 border-black">
+            <div className="relative flex h-16 items-center justify-between">
+              <div className="absolute inset-y-0 left-0 flex items-center sm:hidden">
+                {/* Mobile menu button*/}
+                <Disclosure.Button className="inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white">
+                  <span className="sr-only">Open main menu</span>
+                  {open ? (
+                    <XMarkIcon className="block h-6 w-6" aria-hidden="true" />
+                  ) : (
+                    <Bars3Icon className="block h-6 w-6" aria-hidden="true" />
+                  )}
+                </Disclosure.Button>
+              </div>
+              <div className="flex flex-1 items-center justify-center sm:items-stretch sm:justify-start">
+                <div className="flex flex-shrink-0 items-center">
+                  <div className="h-9 w-10 lg:block">
+                    <Image src={logoMustache} alt="" />
+                  </div>
+                  <div className="h-9 w-14 lg:block">
+                    <Image src={logoWordsPurple} alt="" />
+                  </div>
+                </div>
+                <div className="hidden sm:ml-6 sm:block">
+                  <div className="flex space-x-4">
+                    {navigation.map((item) => (
+                      <a
+                        key={item.name}
+                        href={item.href}
+                        className={classNames(
+                          item.current
+                            ? "bg-[#9187F4] text-white  hover:bg-[#FFC200] hover:text-black"
+                            : "text-purple-dark hover:bg-purple-light hover:text-white",
+                          "px-3 py-2 rounded-3xl text-m font-medium "
+                        )}
+                        aria-current={item.current ? "page" : undefined}
+                      >
+                        {item.name}
+                      </a>
+                    ))}
+                  </div>
+                </div>
+              </div>
+              <div className="absolute inset-y-0 right-0 flex items-center pr-2 sm:static sm:inset-auto sm:ml-6 sm:pr-0">
+                <button
+                  type="button"
+                  className="rounded-full bg-gray-800 p-1 text-gray-400 hover:text-white focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-gray-800"
+                >
+                  <span className="sr-only">View notifications</span>
+                  <StarIcon
+                    className="h-6 w-6 fill-yellow"
+                    aria-hidden="true"
+                  />
+                </button>
+
+                {/* Profile dropdown */}
```

Comment:
instead of these types of sections, split up the return statement into several parts. See this link for more information

Reply:
In this case, it is better to split this up into several comments and put them in different files

Reply:
i cant see the link abhi